pkg/u-boot/Dockerfile

```diff
@@ -8,7 +8,9 @@ SHELL ["/bin/bash", "-eo", "pipefail", "-c"]
 ENV VERSION v2024.10-rc2
 ENV SOURCE_URL https://github.com/u-boot/u-boot/archive/${VERSION}.tar.gz
 ENV RASPBERRY_FIRMWARE_BLOBS_VERSION 1.20211007
+ENV RASPBERRY_FIRMWARE_BLOBS_VERSION2 1.20240306
 ENV RASPBERRY_FIRMWARE_BLOBS https://github.com/raspberrypi/firmware/raw/${RASPBERRY_FIRMWARE_BLOBS_VERSION}
+ENV RASPBERRY_FIRMWARE_BLOBS2 https://github.com/raspberrypi/firmware/raw/${RASPBERRY_FIRMWARE_BLOBS_VERSION2}
```

Comment:
So, are we going to fix the variable names?...

Reply:
Suggestion: `RASPBERRY_5_FIRMWARE_BLOBS`, `RASPBERRY_FIRMWARE_BLOBS_RPI5` or something like that....